03week/ticTacToe.js

```diff
@@ -24,23 +24,47 @@ function printBoard() {
 }
 
```

Comment:
In all your win check functions, it would nice to explicitly return false if the conditions are not met.